test/command_callback/test_golangserver_command_callback.vader

```diff
@@ -0,0 +1,25 @@
+
+Before:
+  call ale#assert#SetUpLinterTest('go', 'golangserver')
+  let sep = has('win32') ? ';' : ':'
+  let $GOPATH=ale#path#Simplify(g:dir . '/go_paths/go1')
```

Comment:
Write `Save $GOPATH` above, so the previous value of `GOPATH` will be restored after running these tests.